MixItUp.WPF/Controls/Chat/ChatUserControl.xaml.cs

```diff
@@ -14,31 +16,61 @@ public partial class ChatUserControl : UserControl
     {
         public UserViewModel User { get; private set; }
 
+        public ChatUserControl()
+        {
+            this.DataContextChanged += ChatUserControl_DataContextChanged;
```

Comment:
I would put these after the InitalizeComponent();

Reply:
Sounds good